src/app/containers/ProposalPage/Ratio.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react';
+
+import { BoxPanel, Wrapper, List, ListItem, RatioValue } from 'app/components/newproposal/RatioInput';
+import { Typography } from 'app/theme';
+
+import { PROPOSAL_CONFIG } from '../NewProposalPage/constant';
+
+interface RatioProps {
+  proposalType: string;
+  proposedList: [];
+}
+
+export default function Ratio({ proposalType, proposedList }: RatioProps) {
+  const [ratioValues, setRatioValues] = React.useState<Array<RatioValue> | undefined>();
+
+  useEffect(() => {
+    (async () => {
+      const result = await PROPOSAL_CONFIG[proposalType].fetchInputData();
+
+      setRatioValues(result);
+    })();
+  }, []);
```

Comment:
`  Line 22:6:  React Hook useEffect has a missing dependency: 'proposalType'. Either include it or remove the dependency array  react-hooks/exhaustive-deps`